lib/fido/views/add_fingerprint_dialog.dart

```diff
@@ -193,35 +211,47 @@ class _AddFingerprintDialogState extends ConsumerState<AddFingerprintDialog>
                     },
                   ),
                 ),
-                LinearProgressIndicator(value: progress),
-                Padding(
-                  padding: const EdgeInsets.all(8.0),
-                  child: Text(_getMessage()),
-                ),
+                if (_fingerprint == null) ...[
```

Comment:
Is the `...[ ]` needed here?

Reply:
Good catch, you're right! I pushed a commit that removes the spread operator (`...`) at this and another place.